### DISCUSSION TITLE
Git LFS shipment with Git

### DISCUSSION THREAD

**Discussion by SalmaFarghaly** [git-lfs/git-lfs#4949] [Q&A] [answered]
Hello,

I have some questions regarding Git LFS, I would appreciate your help very much.
1) Does Git LFS shipped with Git on windows in **all** Git versions ?
2) If I have git version that has Git LFS shipped with it, and then I installed another Git LFS version, Is the Git LFS version that will be used later is the one that I have written in the PATH ?
3) Is Linux or any another platform other than Windows has Git LFS shipped with Git ?

Thanks.

**Comment by chrisd8088** [comment, marked as answer]
Hi!  Briefly, here are some answers, I hope:
1. No, Git LFS is an independent [project](https://git-scm.com/), and is not included with Git by default.  The Git for Windows [project](https://gitforwindows.org/) packages them both together, but that is itself a separate project from either Git or Git LFS.
2. I think it will depend on where you've installed the newer Git LFS.  You can always check which one you're seeing with `git lfs version`, and if you're on Windows, something like `where git-lfs.exe` should tell you which one is being found first in your `PATH`.
3. No, on Linux and macOS the Git LFS project is installed separately from Git.  This could also be true on Windows if you build the projects independently.  Only if you're using Git for Windows are they collected together.
